doc/holochain_101/src/conductors.md

```diff
@@ -1 +1,29 @@
-# [E] Conductors
+# Running Holochain Apps: Conductors
+
+To introduce Conductors, it is useful to zoom out for a moment to the level of how Holochain runs on devices.
+
+Because Holochain is intended to be highly platform and system compatible, the core logic is written in such a way that it can be included into many different codebases. Think of MacOSX, Linux, Windows, Android, iOS, and more. Thus, the core of Holochain was made to be simply a library that gets included in another wrapper which mounts, executes and manages it. Because filling this new need is such a foundational aspect of Holochain, it has its' own name: *Conductor*.
```

Comment:
```suggestion
Holochain was designed to be highly platform and system compatible.  The core logic that runs a DNA instance was written in such a way that it could be included into many different codebases as a library, thus making it easier to build different implementations on the same platform as well as across platforms (MacOSX, Linux, Windows, Android, iOS, and more). Architecturally, Holochain DNAs are intended to be small composible units that provide bits of distributed data integrity functionality.  Thus most Holochain based applications will actually be assemblages of many "bridged" DNA instances.  For this to work we needed a distinct layer that orchestrates the data flow (i.e. zome function call requests and responses), between the transport layer (i.e. HTTP, Websockets, Unix domain sockets, etc) and the DNA instances.  We call the layer that performs these two crucial functions, the *Conductor*, and we have written a `conductor_api` library to make it easy to build actual Conductor implementations.
```